src/viam/components/board/service.py

```diff
@@ -139,7 +139,8 @@ async def ReadAnalogReader(self, stream: Stream[ReadAnalogReaderRequest, ReadAna
             raise e.grpc_error
         timeout = stream.deadline.time_remaining() if stream.deadline else None
         value = await analog_reader.read(extra=struct_to_dict(request.extra), timeout=timeout, metadata=stream.metadata)
-        response = ReadAnalogReaderResponse(value=value)
+        response = ReadAnalogReaderResponse(value=value.value, min_range=value.min_range,
+                                            max_range=value.max_range, step_size=value.step_size)
         await stream.send_message(response)
```

Comment:
You should be able to just do `stream.send_message(value)` now that `value` is of type `ReadAnalogReaderResponse`

Reply:
Sorry to make you catch all these things that I should have caught myself. I appreciate it; I think this version of the code looks way better than the original!